presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -663,6 +665,20 @@ public void testShowPartitions()
     }
 
     @Test
+    public void testCreateTable()
+            throws Exception
+    {
+        assertStatement("CREATE TABLE foo (a VARCHAR, b BIGINT)",
+                new CreateTable(QualifiedName.of("foo"),
+                        ImmutableList.of(new TableElement("a", "VARCHAR"), new TableElement("b", "BIGINT")),
+                        false));
+        assertStatement("CREATE TABLE IF NOT EXISTS bar (c TIMESTAMP)",
```

Comment:
Add a test that validates that `create table if not exists` completes without an error if the table exists, but leaves the existing table untouched.